changelog.md

```diff
@@ -3,6 +3,12 @@ Read more in forum threads:
 * [More GEAR - Improved Emulation of the Propeller](http://forums.parallax.com/showthread.php/100380-More-GEAR-Improved-Emulation-of-the-Propeller)
 * [GEAR: Propeller Debugging Environment](http://forums.parallax.com/showthread.php/91084-GEAR-Propeller-Debugging-Environment)
 
+## V15.03.27
```

Comment:
The version number should be 20.04.23 (following the format <year>.<month>.<day>).

Reply:
I don't see any other problem. At least it runs OK the CTR.spin below. 

`CON

  _CLKMODE = XTAL1 + PLL16X
  _XINFREQ = 5_000_000


OBJ

'  term : "tv_terminal"     'definición original, cambiada a tv_text para usar menos memoria
  term : "tv_text"

VAR

  long  ctr, frq

  
PUB Go | freq

  '' Synthesize frequencies on pin 0 and pin 1
  '' also show FRQ values on tv

  term.start(12)                     'start tv terminal
  
  SynthFreq(0, 1_000_000)            'determine ctr and frq for pin 0
  CTRA := ctr                        'set CTRA
  FRQA := frq                        'set FRQA                   
  DIRA[0]~~                          'make pin output

  term.str(string("CTRA = "))        'show CTRA value
  term.hex(ctr,8)                    
  term.str(string("  FRQA = "))      'show FRQA value
  term.hex(frq,8)                    

  SynthFreq(1, 1_000_001)            'determine ctr and frq for pin1
  CTRB := ctr                        'set CTRB
  FRQB := frq                        'set FRQB                   
  DIRA[1]~~                          'make pin output

  term.str(string(13, "CTRB = "))    'show CTRB value
  term.hex(ctr,8)                    
  term.str(string("  FRQB = "))      'show FRQB value
  term.hex(frq,8)                    

  repeat                             'loop forever to keep cog alive
    waitcnt(0)                       'waitcnt to reduce power


PRI SynthFreq(Pin, Freq) | s, d

'' Determine CTR settings for synthesis of 0..128 MHz in 1 Hz steps
''
'' in:    Pin = pin to output frequency on
''        Freq = actual Hz to synthesize
''
'' out:   ctr and frq hold CTRA/CTRB and FRQA/FRQB values
''
''   Uses NCO mode %00100 for 0..499_999 Hz
''   Uses PLL mode %00010 for 500_000..128_000_000 Hz
''

  Freq := Freq #> 0 <# 128_000_000     'limit frequency range
  
  if Freq < 500_000                    'if 0 to 499_999 Hz,
    ctr := constant(%00100 << 26)      '..set NCO mode
    s := 1                             '..shift = 1

  else                                 'if 500_000 to 128_000_000 Hz,
    ctr := constant(%00010 << 26)      '..set PLL mode
    d := >|((Freq - 1) / 1_000_000)    'determine PLLDIV
    s := 4 - d                         'determine shift
    ctr |= d << 23                     'set PLLDIV
    
  frq := fraction(Freq, CLKFREQ, s)    'Compute FRQA/FRQB value
  ctr |= Pin                           'set PINA to complete CTRA/CTRB value


PRI fraction(a, b, shift) : f

  if shift > 0                         'if shift, pre-shift a or b left
    a <<= shift                        'to maintain significant bits while 
  if shift < 0                         'insuring proper result
    b <<= -shift
 
  repeat 32                            'perform long division of a/b
    f <<= 1
    if a => b
      a -= b
      f++           
    a <<= 1    
`